FirebaseAdmin/FirebaseAdmin/AppCheck/AppCheckToken.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace FirebaseAdmin.Check
+{
+    /// <summary>
+    /// Interface representing an App Check token.
+    /// </summary>
+    /// <remarks>
+    /// Initializes a new instance of the <see cref="AppCheckToken"/> class.
+    /// </remarks>
+    /// <param name="tokenValue">Generator from custom token.</param>
```

Comment:
Is this description correct? It is not matching the description of the property.

Reply:
I have updated the AppCheckToken.cs file, and the issue has been resolved